### ISSUE TITLE
bakeit doesn't handle '413 Request Entity Too Large' well

### ISSUE THREAD

**Issue by efine** [skorokithakis/bakeit#1]
I tried to pipe a 184 KB clipboard buffer through `bakeit` and got the traceback shown below. When I tried to paste it directly into `pastery`, it displayed '413 Request Entity Too Large'. I assume this is the error that is upsetting `bakeit`. It would be nice if `bakeit` displayed a meaningful error message.

~~~
Traceback (most recent call last):
  File "/usr/local/bin/bakeit", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/bakeit/cli.py", line 76, in main
    max_views=args.max_views,
  File "/usr/local/lib/python2.7/dist-packages/bakeit/uploader.py", line 34, in upload
    rd = response.json()
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 808, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python2.7/dist-packages/simplejson/__init__.py", line 505, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.scanner.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
~~~

**Comment by skorokithakis** [comment]
Oops, thanks for that, will fix immediately. I also didn't get a Github notification for this, very odd.

**Comment by skorokithakis** [comment]
Should be fixed in v0.1.8, thanks!
